pdns/syncres.hh

```diff
@@ -720,6 +720,7 @@ public:
   static uint8_t s_ecsipv6limit;
   static uint8_t s_ecsipv4cachelimit;
   static uint8_t s_ecsipv6cachelimit;
+  static unsigned int s_ecscachelimitttl;
```

Comment:
Just a nice but it would be nice to move the new field with the other `unsigned int` ones.

Reply:
> It looks like this setting prevents the insertion of an ECS-specific entry into the cache regardless of the actual scope. I think what we want is to prevent the entry from entering the cache if the scope is greater or equal to `ecs-ipv{4,6}-cache-bits` **AND** the TTL is lower than `ecs-cache-limit-ttl`, and unless I'm mistaken this PR does prevent it if the scope is greater or equal to `ecs-ipv{4,6}-cache-bits` **OR** the TTL is lower than `ecs-cache-limit-ttl`.

Yes, alreay determined myself that the condition is not correct, will revisit.